Sources/Controllers/Help/OAHelpViewController.mm

```diff
@@ -0,0 +1,366 @@
+       }];
+    
+    [dataArr addObject:
+     @{
+       @"name" : @"facebook",
+       @"title" : OALocalizedString(@"facebook"),
+       @"description" : @"https://www.facebook.com/osmandapp",
+       @"type" : kLinkExternalType
+       }];
+    
+    _followData = [NSArray arrayWithArray:dataArr];
+    [dataArr removeAllObjects];
+    
+    [self.tableView reloadData];
+    
+}
+
+- (NSDictionary *) getItem:(NSIndexPath *)indexPath
+{
+    long section = indexPath.section;
+    switch (section) {
+        case firstStepsIndex:
+            return _firstStepsData[indexPath.row];
+        case featuresIndex:
+            return _featuresData[indexPath.row];
+        case pluginsIndex:
+            return _pluginsData[indexPath.row];
+        case otherIndex:
+            return _otherData[indexPath.row];
+        case followIndex:
+            return _followData[indexPath.row];
+        default:
+            return nil;
+    }
+}
+
+#pragma mark - UITableViewDataSource
+
+- (NSInteger) numberOfSectionsInTableView:(UITableView *)tableView
+{
+    return groupCount;
+}
+
+- (NSInteger) tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section
+{
+    if (section == firstStepsIndex)
+        return 3;
+    else if (section == featuresIndex)
+        return 4;
+    else if (section == pluginsIndex)
+        return 6;
+    else if (section == otherIndex)
+        return 5;
+    else if (section == followIndex)
+        return 2;
+    return 0;
+}
+
+- (UITableViewCell *) tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath
+{
+    NSDictionary *item = [self getItem:indexPath];
+    
+    OAIconTextDescCell *cell = (OAIconTextDescCell *)[tableView dequeueReusableCellWithIdentifier:@"OAIconTextDescCell"];
+    if (cell == nil)
+    {
+        NSArray *nib = [[NSBundle mainBundle] loadNibNamed:@"OAIconTextDescCell" owner:self options:nil];
+        cell = (OAIconTextDescCell *)[nib objectAtIndex:0];
+        cell.textView.numberOfLines = 0;
+        cell.textView.autoresizingMask = UIViewAutoresizingFlexibleHeight | UIViewAutoresizingFlexibleWidth;
+        cell.descView.autoresizingMask = UIViewAutoresizingFlexibleHeight | UIViewAutoresizingFlexibleWidth;
+    }
+    
+    if (cell)
+    {
+        CGRect f = cell.textView.frame;
+        f.origin.y = (item[@"description"] == nil || [item[@"description"] length] == 0) ? 10.0 : 0.0;
+        f.size.height = cell.frame.size.height - 20.0;
+        cell.textView.frame = f;
+        
+        cell.contentView.backgroundColor = [UIColor whiteColor];
+        cell.arrowIconView.hidden = YES;
+        [cell.textView setTextColor:[UIColor blackColor]];
+        [cell.textView setText:item[@"title"]];
+        [cell.descView setText:item[@"description"]];
+        [cell showImage:NO];
+    }
+    return cell;
+}
+
+- (CGFloat) tableView:(UITableView *)tableView heightForRowAtIndexPath:(NSIndexPath *)indexPath
+{
+    return 50.0;
```

Comment:
Are you sure 50 is enough? What if text will not fit this height? Also method for estimating row's height is a must.